cs3client/file.py

```diff
@@ -158,23 +168,30 @@ def touch_file(self, auth_token: tuple, resource: Resource) -> None:
         self._status_code_handler.handle_errors(res.status, "touch file", resource.get_file_ref_str())
         self._log.debug(f'msg="Invoked TouchFile" trace="{res.status.trace}"')
 
-    def write_file(self, auth_token: tuple, resource: Resource, content: str | bytes, size: int) -> None:
+    def write_file(
+            self, auth_token: tuple, resource: Resource, content: str | bytes, size: int, lock_md: tuple = None
+    ) -> None:
         """
         Write a file using the given userid as access token. The entire content is written
         and any pre-existing file is deleted (or moved to the previous version if supported),
         writing a file with size 0 is equivalent to "touch file" and should be used if the
         implementation does not support touchfile.
 
         :param auth_token: tuple in the form ('x-access-token', <token>) (see auth.get_token/auth.check_token)
-        :param resource: Resource to write content to.
+        :param resource: Resource to write content to
         :param content: content to write
         :param size: size of content (optional)
+        :param lock_md: tuple (<app_name>, <lock_id>)
         :return: None (Success)
         :raises: FileLockedException (File is locked),
         :raises: AuthenticationException (Authentication failed)
         :raises: UnknownException (Unknown error)
 
         """
+        if lock_md:
+            app_name, lock_id = lock_md
+        else:
+            app_name = lock_id = ''
```

Comment:
Why not making the default value in the params a tuple of empty strings? (also, why empty strings? none is not valid?)